lingpy/basic/wordlist.py

```diff
@@ -1044,7 +1044,35 @@ def coverage(self, stats='absolute'):
             return sum([a / self.height for a in cov.values()]) / self.width
 
     @classmethod
-    def from_cldf(cls, path, columns=[], filter=lambda row: row["Form"], *args, **kwargs):
+    def from_cldf(
+            cls, 
+            path, 
+            columns=[
+                'parameter_id',
+                'concept_name',
+                'language_id',
+                'language_name',
+                'value',
+                'form',
+                'segments',
+                'language_glottocode',
+                'concept_concepticon_id',
+                'language_latitude',
+                'language_longitude',
+                'cognacy'
+                ], 
+            namespace= {
```

Comment:
See above: mutable default args are risky.

Reply:
and here something like a tuple of pairs?

Reply:
Then it's good we cover this anyway, as it was columns = [] before. But now I'll allow for "columns='default'" vs. columns=None.

Reply:
can you give a quick example for tuple of pairs?

Reply:
```python
namespace=(('key', 'value'), ('key2', 'value2'), ...)
```